### ISSUE TITLE
Commands are missing after upgrade

### ISSUE THREAD

**Issue by nielsek** [terminal-discord/weechat-discord#82]
I've upgraded to the latest weechat-discord, then had to upgrade weechat due to the API version mismatch and now weechat is not connecting to discord.

`/discord token $token` - this works fine, and it updates the `.weechat/weecord.conf`

~~~
17:55   weechat     | discord: Updated Discord token
~~~

`/discord connect` - not working

~~~
17:55   weechat     | error: Found argument 'connect' which wasn't expected, or isn't valid in this context
17:55   weechat     | 
17:55   weechat     | If you tried to supply `connect` as a PATTERN use `-- connect`
17:55   weechat     | 
17:55   weechat     | USAGE:
17:55   weechat     |     discord <SUBCOMMAND>
17:55   weechat     | 
17:55   weechat     | For more information try --help
~~~

`/discord autostart` - not working

~~~
17:55   weechat     | error: Found argument 'autostart' which wasn't expected, or isn't valid in this context
17:55   weechat     | 
17:55   weechat     | If you tried to supply `autostart` as a PATTERN use `-- autostart`
17:55   weechat     | 
17:55   weechat     | USAGE:
17:55   weechat     |     discord <SUBCOMMAND>
17:55   weechat     | 
17:55   weechat     | For more information try --help
~~~

`/discord help` - working but it looks like commands are missing

~~~
17:56   weechat     | /discord
17:56   weechat     | 
17:56   weechat     | USAGE:
17:56   weechat     |     discord <SUBCOMMAND>
17:56   weechat     | 
17:56   weechat     | FLAGS:
17:56   weechat     |     -h, --help    Prints help information
17:56   weechat     | 
17:56   weechat     | SUBCOMMANDS:
17:56   weechat     |     channel
17:56   weechat     |     debug
17:56   weechat     |     help       Prints this message or the help of the given subcommand(s)
17:56   weechat     |     server
17:56   weechat     |     token
~~~

I'm running weechat 2.9 and the latest release from Actions.

**Comment by Noskcaj19** [comment]
Woops, that would be my bad.  The link to Github Actions builds is not filtered, so it has builds from all branches.  The build you have is from the still _very_ experimental mk3 branch, which has significant changes to the command system including the removal of the connect command.  I will correct the link to point to the correct branch.

The stable (master) builds can be found [here](https://github.com/terminal-discord/weechat-discord/actions?query=branch%3Amaster).

**Comment by nielsek** [comment]
@Noskcaj19 thanks! It's working now :)

Had to downgrade Weechat to 2.8 too though due to this:

`Error: API mismatch for plugin "~/.weechat/plugins/libweecord.so" (current API: "20200621-01", plugin API: "20200301-03"), failed to load`

**Comment by Noskcaj19** [comment]
The link on the readme has been updated to point to the latest build on the correct branch.

**Comment by Noskcaj19** [comment]
The api mismatch should be fixed in 09b4bca
